neptune/management/internal/api.py

```diff
@@ -433,3 +437,140 @@ def get_workspace_member_list(name: str, api_token: Optional[str] = None) -> Dic
         }
     except HTTPNotFound as e:
         raise WorkspaceNotFound(workspace=name) from e
+
+
+@with_api_exceptions_handler
+def _get_raw_workspace_service_account_list(
+    name: str, api_token: Optional[str] = None
```

Comment:
Can we keep workspace_name and project_name variables in this file?

Reply:
Except for this function, the argument names are a part of product specification :wink:. Leave a comment there if you would like to discuss anything.